Access Rules update (#2)
strongdm/terraform-provider-sdm-examples: README.md

This is the examples repository for the [strongDM Terraform Provider](https://github.com/strongdm/terraform-provider-sdm).

---
> **NOTE:**  

> > To increase flexibility when managing thousands of Resources, Role Grants have

Reviewer: I'd maybe just say large numbers of resources? Surely Access Rules are more flexible with even mediocre numbers, I don't think you have to have thousands to realize the benefits.

> > To increase flexibility when managing thousands of Resources, Role Grants have

Author: ```suggestion
> To increase flexibility when managing a large volume of Resources, Role Grants have
```

been deprecated in favor of Access Rules, which allow you to grant access based

> on Resource Tags and Type.

Author: ```suggestion
on Resource tags and type.
```

>

> > Previously, you would grant a Role access to specific resources by ID via Role

Author: ```suggestion
> Previously, you would grant a Role access to specific Resources by ID via Role
```

> Grants. Now, when using Access Rules, the best practice is to grant Resources access based on Type and Tags.

Reviewer: Same as above

> Grants. Now, when using Access Rules, the best practice is to grant Resources access based on Type and Tags.

Author: ```suggestion
Grants. Now, when using Access Rules, the best practice is to give Roles access to Resources based on type and tags.
```

>

> >The following examples demonstrate Dynamic Access Rules with Tags and Resource Types, as well as Static Access Rules for backwards compatibility with Role Grants. If it is _necessary_ to grant access to specific Resources in the same way as Role Grants did, you can use Resource IDs directly in Access Rules.

Author: ```suggestion
>The following examples demonstrate Dynamic Access Rules with tags and Resource types, as well as Static Access Rules for backwards compatibility with Role Grants. If it is _necessary_ to grant access to specific Resources in the same way as Role Grants did, you can use Resource IDs directly in Access Rules.
```
